packages/renderer/src/lib/help/HelpActions.svelte

```diff
@@ -19,7 +19,11 @@ function handleEscape({ key }: KeyboardEvent): void {
 }
 
 function onWindowClick(e: any): void {
-  showMenu = outsideWindow.contains(e.target);
+  // We listen to the click event on anything BUT the question circle icon which is
+  // where the help menu is. This is to ensure the menu is closed if the user clicks the circle again.
+  if (!e.srcElement.classList.contains('fa-question-circle')) {
+    showMenu = outsideWindow.contains(e.target);
+  }
```

Comment:
issue: the unit test is not covering this part

unit test should create a element in the dom, add the fa-question-circle and click on this element (that should trigger the onWindowClick, etc.

ideally the fa-question-circle should come from the definition of the status bar entry being added for the help menu

b/c let's say tomorrow we change the status bar icon for the help menu from a question-circle to another icon, this unit test will continue to work but the app won't work (as the icon would have changed)

Reply:
> issue: the unit test is not covering this part
> 
> unit test should create a element in the dom, add the fa-question-circle and click on this element (that should trigger the onWindowClick, etc.
> 
> ideally the fa-question-circle should come from the definition of the status bar entry being added for the help menu
> 
> b/c let's say tomorrow we change the status bar icon for the help menu from a question-circle to another icon, this unit test will continue to work but the app won't work (as the icon would have changed)

You are 100% right, this test doesn't cover it at all and that's my mistake. Sorry!

I'll re-write the test and have it inject the fa-question-circle so you can click that and it'll toggle correctly.